sources/packages/backend/workflow/test/2024-2025/parttime-assessment/appeals/parttime-assessment-student-additional-transportation.e2e-spec.ts

```diff
@@ -91,21 +103,8 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-student-addition
       // Assert
       expect(
         calculatedAssessment.variables
-          .calculatedDataEligibleForAnAdditionalTransportationAllowance,
-      ).toBe("no");
-      expect(
-        calculatedAssessment.variables.calculatedDataAdditionalTransportKm,
-      ).toBe(0);
-      expect(
-        calculatedAssessment.variables.calculatedDataAdditionalTransportCost,
-      ).toBe(0);
-      expect(
-        calculatedAssessment.variables.calculatedDataAdditionalTransportWeeks,
-      ).toBe(0);
-      expect(
-        calculatedAssessment.variables
-          .calculatedDataAdditionalTransportPlacement,
-      ).toBe(false);
```

Comment:
Is the expectation now that those variables will not be defined?
Does it make sense to check for it or maybe rename the test's description?